dll/win32/shell32/dialogs/folder_options.cpp

```diff
@@ -1911,6 +1965,14 @@ FolderOptionsFileTypesDlg(
                         FileTypesDlg_AddExt(hwndDlg, newext.szExt, newext.szFileType);
                     }
                     break;
+                case IDC_FILETYPES_DELETE:
+                    LoadStringW(shell32_hInstance, IDS_REMOVE_EXT, Buffer, _countof(Buffer));
```

Comment:
This is a .cpp file, why not use CStringW?
`CStringW Buffer(MAKEINTRESOURCEW(IDS_REMOVE_EXT));`

Reply:
Note: using the ATL CStringW could have been done also in the previous patch that implemented the "create new file extension" dialog.

Reply:
OK.